test/test-source-map-generator.js

```diff
@@ -42,6 +42,13 @@ exports['test adding mappings (case 1)'] = function (assert) {
       generated: { line: 1, column: 1 }
     });
   });
+
+  assert.doesNotThrow(function () {
+    map.addMapping({
+        generated: { line: 1, column: 1 },
+        original: { line: null, column: null }
```

Comment:
My guess is that you have another issue because you ideally want to map to an actual location in an original file.